content/pages/education/opt-out.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This is a placeholder page for the static landing page.

Reply:
My understanding is that this static page isn't connected to the app yet. Do we want to use a widget similar to what's on the `education/apply` [page](https://github.com/department-of-veterans-affairs/vets-website/blame/master/content/pages/education/apply.md#L15-L19)?

Reply:
Using a widget similar to that which is on the `education/apply` page makes sense, if it seems like overkill to display a modal, we can go with a vanilla JS option for now as well. 

Some general steps on the react approach are described below:

Once a widget is set up and ready to use:

 1. Loading logic specifying the appropriate markdown page needs to be added to `static-pages-entry.js`
2. the markdown page needs to include the widget in its frontmatter under `widgets` and an anchor for the react component must be included in the markup